api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -931,6 +931,7 @@ public class ApiConstants {
     public static final String LOGIN = "login";
     public static final String LOGOUT = "logout";
     public static final String LIST_IDPS = "listIdps";
+    public static final String IS_SELECTION_OF_STATIC_NAT_ALLOWED = "selectsnatipallowed";
```

Comment:
I think it is Source Nat IP, not Static Nat Ip, right ?

Reply:
Correct